src/content/docs/en/guides/images.mdx

```diff
@@ -127,242 +129,24 @@ This will require you to update your existing images, remove the current Astro i
-:::
-
-### `<Image />`
-
-Astro's [`<Image />` component](/en/guides/integrations-guide/image/#image-) allows you to optimize a single image and specify width, height, and/or aspect ratio. You can even transform your image to a particular output format.
-
-This component is useful for images where you want to keep a consistent size across displays, or closely control the quality of an image (e.g. logos).
-
-For responsive images, or art direction, use the `<Picture />` component instead.
-
-#### Remote Images
-
-(required attributes: [`src`](/en/guides/integrations-guide/image/#src), [`alt`](/en/guides/integrations-guide/image/#alt), [`format`](/en/guides/integrations-guide/image/#format), and dimensions)
-
-Pass a full URL to the `<Image />` component's `src` attribute and include a value for `alt`.
-
-The `<Image />` component cannot determine the original file format of a remote image, so you must provide an output `format` (e.g. png, avif) to transform your remote image.
-
-You must also either provide [`width`](/en/guides/integrations-guide/image/#width) and [`height`](/en/guides/integrations-guide/image/#height), or one of the dimensions plus an [`aspectRatio`](/en/guides/integrations-guide/image/#aspectratio) to avoid content layout shifts because the `<Image />` component does not know the dimensions of a remote image.
-
-[All other properties](/en/guides/integrations-guide/image/#image-) are optional.
-
-#### Local Images in `src/`
-
-(required attributes: [`src`](/en/guides/integrations-guide/image/#src), and [`alt`](/en/guides/integrations-guide/image/#alt))
-
-Import your image in frontmatter and pass it directly to the `<Image />` component's `src` attribute.
-
-`alt` is required, but [all other properties](/en/guides/integrations-guide/image/#image-) are optional and will default to the image file's original properties if not provided.
-
-#### Images in `public/`
-
-(required attributes: [`src`](/en/guides/integrations-guide/image/#src), [`alt`](/en/guides/integrations-guide/image/#alt), [`format`](/en/guides/integrations-guide/image/#format), and dimensions)
-
-Pass the component's `src` attribute a path relative to the public folder and include a value for `alt`.
-
-It will be treated as a remote image, which requires either both [`width`](/en/guides/integrations-guide/image/#width) and [`height`](/en/guides/integrations-guide/image/#height), or one dimension and an [`aspectRatio`](/en/guides/integrations-guide/image/#aspectratio) attribute.
-
-A value for the `format` attribute (e.g. png, avif) to transform your image is required.
-
-[All other properties](/en/guides/integrations-guide/image/#image-) are optional.
-
-Your original image will be copied unprocessed to the build folder, like all files located in `public/`, and Astro's image integration will also return optimized versions of the image.
+:::note[to be deprecated in v3.0]
+This integration will no longer be actively supported in Astro v3.0. 
```

Comment:
Maybe `@astrojs/image` should be included in this paragraph so people know what integration we're talking about?